lib/ui/window/window.cc

```diff
@@ -13,14 +13,48 @@
 
 namespace flutter {
 
-Window::Window(int64_t window_id, ViewportMetrics metrics)
-    : window_id_(window_id), viewport_metrics_(std::move(metrics)) {
-  library_.Set(tonic::DartState::Current(),
-               Dart_LookupLibrary(tonic::ToDart("dart:ui")));
-}
+Window::Window(tonic::DartPersistentValue& library,
```

Comment:
Is there any chance we can rename this to `View`? If yes, should this be part of this change?

Reply:
We should definitely rename it to `View`. I don't want to do this since this PR is already pretty big. But mostly, we're probably removing `Window`. The only place that uses it is `DrawShadow`. (`RuntimeController` also uses it now but can be migrated.)

Reply:
Sounds good to me, marking as resolved